test/e2e/alerts.js

```diff
@@ -362,38 +353,59 @@ const runScenario = async (client, scenarioId, options) => {
     if (!options.showOutput) {
       console.error(`Re-run with -v to see its output.`)
     }
-    return false
+    throw new Error('Error while running onOperationOrBillCreate')
```

Comment:
The error message includes "onOperationOrBillCreate". Shouldn't it be "alerts"?

Reply:
No here, this is the onOperationOrBillCreate.js service that is launched.